cabal-install/Distribution/Client/CmdConfigure.hs

```diff
@@ -83,9 +85,18 @@ configureAction (configFlags, configExFlags, installFlags, haddockFlags)
     baseCtx <- establishProjectBaseContext verbosity cliConfig
 
     -- Write out the @cabal.project.local@ so it gets picked up by the
-    -- planning phase.
-    writeProjectLocalExtraConfig (distDirLayout baseCtx)
-                                 cliConfig
+    -- planning phase. If old config exists, then print the contents
+    -- before overwriting
+    exists <- doesFileExist "cabal.project.local"
+    if exists
+      then do
+        contents <- readFile "cabal.project.local"
+        putStr contents
```

Comment:
1. I think the `cabal.project.local` file would already contain a newline character at the end. Thus two of those would be redundant. Although this isn't necessary. I think I should probably use putStrLn

2. Yeah! That would be better.